app.js

```diff
@@ -123,5 +123,6 @@ function displayAllGames() {
         const gameEl = renderGame(game.name1, game.name2, game.score1, game.score2);
         pastGamesEl.append(gameEl);
     }
-    
+
+    renderGame(name1, name2, score1, score2);
 }
```

Comment:
Lines 45, 53, 61, 69 - is this necessary? It seems like this may be an extra step considering the refreshCurrentGame() function is doing this work already. 

Lines 79 - 82 - No need for the template literal here since you want the values stored in this object to be the actual values they are in state. With the way you have it written now it is converting all the values to strings, which is not ideal in this case (but doesn't really impact anything in this app).

Reply:
Good catch, thank you for pointing that out. Lines 45, 53, 61, and 69 are vestigial, I must have forgotten to remove them after writing the refreshCurrentGame() function.

Regarding lines 79 - 82, I was still unclear about the values that template literals return, so I tried it both ways, and seeing that nothing had changed, left it as is. I appreciate the explanation. 

Let me make sure I have this right: although the display might appear to show numerical values, enclosing the variables in template literals has converted the numerical data to strings, so we're just seeing a representation of numbers as strings. Like how a picture of a cat is not a cat, although they look similar. Is that close? I can see how that could be a major problem in another application.

Reply:
You got it! What is happening is called **string interpolation**. The ```${}``` is just a placeholder for anything you'd like to convert to a string and inject into the placeholder. Stuff written inside the ```${}``` whether it is a variable ```${myVariable}``` or an expression ```${5 * 5}``` will be evaluated and then converted to a string.

Reply:
Good to know, that seems like a really useful feature. Thanks!